src/world/generator/PopulationTask.php

```
@@ -29,130 +9,91 @@
+    private ?string $centerChunk;
+    private string $adjacentChunksData;
+
+    public function __construct(
+        private int $worldId,
+        private int $chunkX,
+        private int $chunkZ,
+        ?Chunk $chunk,
+        array $adjacentChunks, // array<int, Chunk|null>
+        \Closure $onCompletion  // fn(Chunk, array<int, Chunk>): void
+    ) {
+        $this->centerChunk = $chunk?->serializeTerrain();
```

PHPStan (2.1.15)

Call to an undefined method pocketmine\world\format\Chunk::serializeTerrain().


(method.notFound)